codalab/bin/server.py

```diff
@@ -13,18 +13,18 @@
 class ClFileWatcherEventHandler(FileSystemEventHandler):
     SERVER_PROCESS = None
 
-    def __init__(self, argv):
+    def __init__(self, argv) -> None:
         super(ClFileWatcherEventHandler, self).__init__()
         self.argv = argv
         self.restart()
 
-    def restart(self):
+    def restart(self) -> None:
         if self.SERVER_PROCESS:
             self.SERVER_PROCESS.kill()
 
         self.SERVER_PROCESS = subprocess.Popen(self.argv)
 
-    def on_any_event(self, event):
+    def on_any_event(self, event) -> None:
         extensions_to_watch = ('.js', '.py', '.html', '.css', '.tpl')
```

Comment:
Should we type annotate variables too?

Reply:
@yiboliu It might be too much to type annotate the function body. What do you think about the type annotating the function parameters in this PR, so the function definition is type annotated?